HACKING.md

```diff
@@ -22,6 +22,16 @@ Following terminal commands are useful to deploy `frontend` and `backend`.
 npm run deploy
 ```
 
+**NOTE**: for macOS, you might need to manually install `llvm` and patch `clang` lib path. See example for `zsh` shell:
```

Comment:
In case you like it, you can also use the new alerts (https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) to display a note on GitHub.

Reply:
Thank you! I didn't know about those